cluster/install/fluo-cluster/bin/impl/test.sh

```diff
@@ -37,6 +37,8 @@ git clone -b $TEST_BRANCH $TEST_REPO $FLUO_APP_NAME
 
 cd $FLUO_APP_NAME
 
+$FLUO_HOME/bin/fluo new $FLUO_APP_NAME
```

Comment:
This is a good addition.  How about checking to see if a directory exists for `$FLUO_APP_NAME` first and if it does killing the application and removing the directory before calling `fluo new`